htdocs/core/modules/mailings/contacts1.modules.php

```diff
@@ -408,8 +408,9 @@ public function add_to_target($mailing_id)
 		}
 
 		// Filter on language
+		// we need substring because escape($filter_lang) take just the short value
```

Comment:
Do you lean that $filter_lang is a language code on 2 chars ?
We should receive a language code on 5 chars. May be trouble is before, when getting the filter_lang ?

Reply:
yes when I add a new contact in the section third-part, we have code on 5 chars inserted in DB but when we try to filter with the select $filter_lang , it took just to chars, and when I remove the $filter_lang every thing works fine